Gemfile

```diff
@@ -42,6 +42,10 @@ group :bench do
 end
 
 group :test do
+  # allows for rspec-like invocation of single tests
+  # e.g.: $ m test/path.rb:lineNum
+  gem 'm', '~> 1.4.2'
```

Comment:
It's a critical path, so yes

On Sun, Apr 10, 2016 at 7:27 PM L. Preston Sego III <
notifications@github.com> wrote:

> In Gemfile
> <https://github.com/rails-api/active_model_serializers/pull/1674#discussion_r59144957>
> :
>
> > @@ -42,6 +42,10 @@ group :bench do
> >  end
> >
> >  group :test do
> > +  # allows for rspec-like invocation of single tests
> > +  # e.g.: $ m test/path.rb:lineNum
> > +  gem 'm', '~> 1.4.2'
>
> is it that much a hit to test time to include it? it's a super small gem.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/rails-api/active_model_serializers/pull/1674/files/4c534e28890f0078b6ae8714cab1c516f1d8e283#r59144957>
>